doc-deploy-dev/action.yml

```diff
@@ -15,6 +15,11 @@ inputs:
     description: "Required token for documentation deployment."
     required: true
     type: string
+  use-downloaded-artifact:
```

Comment:
May we renamed this input variable as `skip_artifact_download`? This is similar to the `skip_installation` input variable that we have in other actions.

Reply:
I agree with this. I will also check if it is a good addition to the stable too.

I just hope I'm not customizing a general action such as this!

Reply:
Resolving because we are changing this whole thing